formbricks/formbricks
# chore: move all services into its folder

## What does this PR do?

This PR deletes the `packages/lib/services` folder and moves every file into its own folder in `packages/lib`.

Update the path to the services in "How we code at Formbricks"
<img width="1369" alt="image" src="https://github.com/formbricks/formbricks/assets/31730715/50d5df3b-2fbd-4f03-8504-30198f402558">

## Type of change




## How should this be tested?

- Perform a sanity test around application
- Run `pnpm build` to make sure everything builds properly

## Checklist



### Required


### Appreciated
